pkg/virt-operator/resource/apply/rbac.go

```diff
@@ -228,15 +231,15 @@ func enforceAPIGroup(existing runtime.Object, required runtime.Object) {
 
 	requiredRoleRef.APIGroup = rbacv1.GroupName
 	for i := range requiredSubjects {
-		if existingSubjects[i].Kind == "User" {
+		if requiredSubjects[i].Kind == "User" {
 			requiredSubjects[i].APIGroup = rbacv1.GroupName
 		}
 	}
 }
 
-func arePolicyRulesEqual(role1 runtime.Object, role2 runtime.Object) (equal bool) {
+func changeRbacExistingByRequired(existing runtime.Object, required runtime.Object) (modified bool) {
```

Comment:
nit: I understand that the naming is throughout the code like this, i.e. `existing` vs. `required`, I find the wording still a bit confusing as what we are doing here is reconcile the current to a desired state, which is why I would name the method 
```
reconcileRBAC(current, desired runtime.Object) (modified bool) {
```

Reply:
I agree, your terminology is better. Thanks!